.ycm_extra_conf.py

```diff
@@ -0,0 +1,211 @@
+# This file is NOT licensed under the GPLv3, which is the license for the rest
+# of YouCompleteMe.
```

Comment:
I see  .ycm_extra_conf.py, please remove that

Reply:
Jesus. I am sorry. Done.